### DISCUSSION TITLE
[QMP iteración 4] Error en la API de AccuWeather

### DISCUSSION THREAD

**Discussion by Damuchi99** [dds-jv/foro#130] [Consultas de objetos] [answered]
Buenas tardes, resulta que al extraer la API de AccuWeather al proyecto me tira un error que me dice que el tipo de retorno tiene que ser de List<HashMap>, quisiera saber si hay algo mal o a que se debe este error.
Adjunto una captura con el correspondiente error.
Desde ya muchas gracias y espero su respuesta.
![image](https://user-images.githubusercontent.com/38899106/82949560-36a79400-9f7a-11ea-8ee7-e35e07ca8a6c.png)

**Comment by roli93** [comment, marked as answer]
@Damuchi99 Mmm, es raro que te tire un error en vez de un warning. De todas formas, ahí le agregué los parámetros de tipo para que no se queje más. Avisame si anduvo

**Comment by Damuchi99** [comment]
Ahí me anduvo, muchas gracias!
